haroldtreen/readmeio-sync
# Remote clean functionality + Documentation

# Remote Cleaning of Readme.io Projects + Documentation

Adds a CLI command for deleting all remote content which is not listed in the local `syncRegistry.json`. Finishes basic documentation.

Status: **Ready for Review**

Reviewers: @donnielrt @fractaltheory 
JIRA: [RTM-233](https://mobify.atlassian.net/browse/RTM-233)
## Changes
- Adds `readmeio-sync clean-remote` command
- Completes documentation in Readme.md
## Test Drive
- Make sure you have Readme.io credentials set as environment variables (`README_EMAIL` & `README_PASSWORD`).
- Configure a remote to sync with using `readmeio-sync config` (you can use the same project for staging and production)
- Initialize with `readmeio-sync init`
- Go into the created `syncRegistry.json` and delete a custom page/ or document object.
- Run `readmeio-sync clean-remote`
- Verify that the content deleted from the registry has been deleted in Readme.io
- `grunt test`
